site/src/components/RichParameterInput/RichParameterInput.tsx

```diff
@@ -102,6 +116,23 @@ const ParameterLabel: FC<ParameterLabelProps> = ({ parameter }) => {
     ? parameter.display_name
     : parameter.name;
 
+  const labelPrimary = (
+    <span css={styles.labelPrimary}>
+      {displayName}
+
+      {!parameter.required && (
+        <Tooltip title="If no value is specified, the system will default to the value set by the administrator.">
+          <span css={styles.optionalLabel}>(optional)</span>
+        </Tooltip>
+      )}
+      {!parameter.mutable && (
+        <Tooltip title="This value cannot be modified after the workspace has been created.">
+          <span css={styles.immutableLabel}>Immutable*</span>
```

Comment:
Why is there an asterisk next to this label?

Reply:
fyi: design updated